ideascube/mediacenter/models.py

```diff
@@ -38,16 +38,16 @@ class Document(SearchMixin, TimeStampedModel):
     APP = 'app'
     OTHER = 'other'
 
-    KIND_CHOICES = (
-        (IMAGE, _('image')),
-        (AUDIO, _('sound')),
-        (VIDEO, _('video')),
-        (PDF, _('pdf')),
-        (TEXT, _('text')),
-        (EPUB, _('epub')),
-        (APP, _('app')),
-        (OTHER, _('other')),
-    )
+    KIND_CHOICES = {
```

Comment:
I think we'd better keep a tuple as `KIND_CHOICES` and have a dict generated from it:

`KIND_DICT = dict(KIND_CHOICES)`

This for two reasons:
- be convention, we expect something named `XXX_CHOICES` to be a tuple
- more important: when using a dict, the order of the generated select will be random, which is not user friendly ;)